motoman_hc20_support/package.xml

```diff
@@ -7,26 +7,43 @@
     <p>ROS-Industrial support for the Motoman HC20 (and variants).</p>
     <p>
       This package contains configuration data, 3D models and launch files
-      for Motoman HC20 manipulators. This currently includes the base model
-      which also supports the DT and XP variants.
+      for Motoman HC20, HC20DTP and HC30PL manipulators. The HC20 base model
+      also supports the DT and XP variants.
     </p>
     <p>
       <b>Specifications</b>
     </p>
     <ul>
-      <li>HC20 - Default</li>
-      <li>HC20DT - Default</li>
-      <li>HC20XP - Default</li>
+      <li>HC20</li>
+      <li>HC20DT - Same as HC20</li>
+      <li>HC20XP - Same as HC20</li>
+      <li>HC20DTP</li>
+      <li>HC30PL</li>
    </ul>
     <p>
-      Joint limits and maximum joint velocities are based on the information 
-      found in <em>YASKAWA MOTOMAN-HC20DT INSTRUCTIONS (HW1486129)</em>
-      version <em>190085-1CD, rev 1</em>.
+      Joint limits and maximum joint velocities are based on the information
+      found in: <em>YASKAWA MOTOMAN-HC20DT INSTRUCTIONS (HW1486129)</em>
+      version <em>190085-1CD, rev 1</em>,
+      <em>YASKAWA MOTOMAN-HC20DTP INSTRUCTIONS (HW2480394)</em> version
+      <em>193888-1CD, rev 3</em> and
+      <em>YASKAWA MOTOMAN-HC30PL INSTRUCTIONS (HW2480886)</em> version
+      <em>196230-1CD, rev 2</em>.
     </p>
     <p>
-      All urdfs are based on the default motion and joint velocity limits, 
+      All urdfs are based on the default motion and joint velocity limits,
       unless noted otherwise.
     </p>
+    <p>
+      NOTE: the HC30PL is the palletizing variant of the HC20 where the wrist
+      joints ('joint_4_r' and 'joint_5_b') are limited to +/-15 degrees with
+      respect to the ground plane normal.
```

Comment:
I understand this constraint to mostly limit the wrist / tool0 / flange orientation as a combination of joint_4_r and joint_5_b.

```suggestion
      NOTE: the HC30PL is the palletizing variant of the HC20 where the wrist
      joints ('joint_4_r' and 'joint_5_b') are constrained to orient the flange to 
      within +/-15 degrees with respect to the ground plane normal.
```

Reply:
given the zero pose of those two joints is aligned with the ground plane normal, would there be a difference?

I'm also not sure we can orient a surface in the way you describe. We'd either have to align the two normals -- ground plane and flange normal -- or say something about the flange surface having to be orthogonal to the ground plane normal, which isn't really clear.

Reply:
You're right, I clarified that the `flange normal` is to be aligned to the ground plane normal.

```suggestion
      NOTE: the HC30PL is the palletizing variant of the HC20 where the wrist
      joints ('joint_4_r' and 'joint_5_b') are constrained to orient the flange normal 
      to within +/-15 degrees with respect to the ground plane normal.
```

Reply:
as I wrote, as the zero pose of the joints already causes the flange normal to be aligned with the ground plane normal, I believe it'd be ok to say something about the joint limits -- which is easier to understand I believe than surface normals.

Reply:
Ok. We'll merge it then.